README.md

```diff
@@ -10,7 +10,6 @@
 celestia-core is a fork of [tendermint/tendermint](https://github.com/tendermint/tendermint) with the following changes:
 
 1. Early adoption of the ABCI++ methods: `PrepareProposal` and `ProcessProposal` because they haven't yet landed in a Tendermint release.
-1. Modifications to how `DataHash` in the block header is determined. In Tendermint, `DataHash` is based on the transactions included in a block. In celestia-core, `DataHash` was modified to include the Merkle root of the row and column roots of the erasure coded data square. See [ADR 008](https://github.com/celestiaorg/celestia-core/blob/v0.34.x-celestia/docs/celestia-architecture/adr-008-updating-to-tendermint-v0.35.x.md?plain=1#L20) for the motivation or [./pkg/da/data_availability_header.go](./pkg/da/data_availability_header.go) for the implementation.
```

Comment:
don't we still do this? Perhaps we should just update it to point to the app's code for pkg/da or ADR001

Reply:
> It seems like we still modify a block header's DataHash but I assumed that moving the da package from celestia-core => celestia-app implied that a celestia-core modification was no longer necessary.

most of celestia-core is just so that we can enable the application to be able to change data root, so while we might want to reword the comment a bit, I definitely think we should mention it.

>So I would expect some step in-between where celestia-core takes the blockData.Hash from prepare_proposal and propagates it into the header's DataHash but I can't find such a modification in https://github.com/rootulp/celestia-core/pull/59? Do you know where that might occur?

its a tad sneaky, but we pass it via the block data. a minor alternative to doing that would be to pass it via the response from prepare proposal, and then stick it in the block data, but keeping it in the block data (where it has to end up anywho) make sense to me. wdyt?

https://github.com/celestiaorg/celestia-core/blob/2ec23f804691afc196d0104616e6c880d4c1ca41/types/block.go#L1041-L1043

Reply:
Wow that is sneaky and based on the diff: https://github.com/rootulp/celestia-core/pull/59/files#r1035009309 it looks like this would work with vanilla Tendermint too. I'll rephrase this line in the README

Reply:
Resolved in https://github.com/celestiaorg/celestia-core/pull/895/commits/50e9b8a57d8ff2822baa178ff88cef00496c59b9

Reply:
Lol that is sneaky. 

Would it make sense to remove https://github.com/celestiaorg/celestia-core/blob/2ec23f804691afc196d0104616e6c880d4c1ca41/types/block.go#L1037-L1039  then because we would never want this to happen right?